cmd/crc/cmd/daemon.go

```diff
@@ -217,6 +218,10 @@ func run(configuration *types.Configuration) error {
 			if err != nil && !errors.Is(err, net.ErrClosed) {
 				logging.Errorf("unixgramListener error: %v", err)
 			}
+			if errors.Is(err, drivers.ErrNotImplemented) {
```

Comment:
Do we want the `logging.Errorf("unixgramListener error: %v", err)` message to trigger in this situation?